price-feeder/oracle/provider/binance.go

```diff
@@ -217,7 +217,7 @@ func (p *BinanceProvider) getTickerPrice(key string) (TickerPrice, error) {
 
 	ticker, ok := p.tickers[key]
 	if !ok {
-		return TickerPrice{}, fmt.Errorf("binance provider failed to get ticker price for %s", key)
+		return TickerPrice{}, fmt.Errorf("binance failed to get ticker price for %s", key)
```

Comment:
I thought about moving these all into one type in `providers.go` but the messages seem to be a bit different per case, wdyt @RafilxTenfen ?

Reply:
atm seems fine to let this way, maybe later we could define a pattern for these messages in the providers